docs/S03-smart-contracts/M4-design-patterns/L4-oracles/index.md

```diff
@@ -8,11 +8,11 @@ Additionally, what if we'd like to interact with external computation in the out
 
 Such data and external computation exists but they are beyond the _trust boundary_ established by our blockchain protocols.
 
-One way blockchain developers have tried to bridge the gap is through what are called **off-chain [oracles.](https://en.wikipedia.org/wiki/Blockchain_oracle){target=_blank}** Off-chain oracles are agents that find and verify real-world information and submit them to the blockchain to be used by smart contracts. They can trigger smart contract executions when the data is obtained or predefined conditions are meet (e.g. time, weather, tracking, payments). Off-chain oracles are provided by organizations like [Provable Things,](https://provable.xyz/){target=_blank} [Chainlink Labs,](https://chain.link/){target=_blank} and more.
+One way blockchain developers have tried to bridge the gap is through what are called **off-chain [oracles.](https://en.wikipedia.org/wiki/Blockchain_oracle)** Off-chain oracles are agents that find and verify real-world information and submit them to the blockchain to be used by smart contracts. They can trigger smart contract executions when the data is obtained or predefined conditions are meet (e.g. time, weather, tracking, payments). Off-chain oracles are provided by organizations like [Provable Things,](https://provable.xyz/{target=_blank}) [Chainlink Labs,](https://chain.link/{target=_blank}) and more.
 
-Please note that DeFi has led to on-chain oracles, particularly dealing with token prices, which we will discuss more in the section on DeFi. Such examples of on-chain oracles are [Chainlink's Data Feeds](https://docs.chain.link/docs/get-the-latest-price/){target=_blank}, Uniswap's `[Observations](https://docs.uniswap.org/protocol/concepts/V3-overview/oracle){target=_blank}`, and MakerDAO's [Feeds](https://developer.makerdao.com/feeds/){target=_blank}.
+Please note that DeFi has led to on-chain oracles, particularly dealing with token prices, which we will discuss more in the section on DeFi. Such examples of on-chain oracles are [Chainlink's Data Feeds](https://docs.chain.link/docs/get-the-latest-price/{target=_blank}), Uniswap's [Observations](https://docs.uniswap.org/protocol/concepts/V3-overview/oracle{target=_blank}), and MakerDAO's [Feeds](https://developer.makerdao.com/feeds/{target=_blank}).
```

Comment:
This one is correct! The rest would actually break our build process. Could you remove them?

Reply:
Yes!